apps/chat-ios/Podfile

```diff
@@ -1,5 +1,5 @@
 # Uncomment the next line to define a global platform for your project
-platform :ios, '13.0'
+# platform :ios, '9.0'
```

Comment:
so what's minimum version we should support? Same as Chime which is iOS 10?

Reply:
CocoaPods provides a default deployment target if one is not specified. The current default values are 4.3 for iOS

Reply:
It was commented before and we do not specify any iOS version. Changed to 13 while testing local build but it did not make any difference.